src/components/navigation/NavBarLinks.js

```diff
@@ -56,12 +50,12 @@ const NavBarLinks = () => {
         </>
       ) : (
         <>
-          <Button href="/register" variant="container">
+          <StyledBtn href="/register" variant="container">
```

Comment:
Nat, can you please change the href to `/login` -- we don't have `/register` anymore since login/register are from one component.. you can test the current one with `localhost:3000/login` to refer to what i mean by one component...

Reply:
actually, please delete the `register` link from the navbar...

Reply:
ok... I will make the changes.